spec/debug.spec.js

```diff
@@ -5,21 +5,76 @@ const runner = require('./runner');
 describe('debug.spec: debug namespace tests', () => {
     describe('debug.log', () => {
         it('logs a simple message', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.debug.log('Hello! This is a simple message');
                 let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
                 await __close(logFileContent);
             `);
             assert.ok(runner.getOutput().includes('Hello! This is a simple message'));
         });
 
+        it('logs an info message', async () => {
+            runner.run(`
+                await Neutralino.debug.log('Hello! This is a info message', 'INFO');
+                let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
+                await __close(logFileContent);
+            `);
+            assert.ok(runner.getOutput().includes('Hello! This is a info message'));
+        });
+
         it('logs an error message', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.debug.log('Hello! This is an error message', 'ERROR');
                 let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
                 await __close(logFileContent);
             `);
             assert.ok(runner.getOutput().includes('Hello! This is an error message'));
         });
+
+        it('logs a warning message', async () => {
+            runner.run(`
+                await Neutralino.debug.log('Hello! This is a warning message', 'WARNING');
+                let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
+                await __close(logFileContent);
+            `);
+            assert.ok(runner.getOutput().includes('Hello! This is a warning message'));
```

Comment:
The `WARNING` option outputs the `WARNING` word into the logfile, so can we add another assertion for it as well?